bloggies_frontend/src/BlogCard/Functions.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react'
+import axios from 'axios'
+
+//Functionality for payment success and for payment cancelation 
+
+
+interface FormData {
+    answer1: string    
+    answer2: string 
+    answer3: string 
+
+}
+
+
+
+
+const formPost = (formData: FormData) => {
+     // goes on register form to post membership registration status
+     return axios.post('/register/membership-status')
```

Comment:
Is it okay for us to continue using the Fetch API?

Reply:
Yes that's fine we can. I guess for the sake of consistency we should all be using fetch if you're more comfortable with it I've used both so I don't have a strong preference either way. I'll make the changes to reflect that. Thanks.